### DISCUSSION TITLE
DJ5 compatibility

### DISCUSSION THREAD

**Discussion by maciej-sitowski** [barseghyanartur/django-elasticsearch-dsl-drf#313] [General]
Hey, 
Any hopes, chances to make it compatible with DJ5 and DRF3.15?

Best regards

**Comment by barseghyanartur** [comment]
Is it incompatible ATM? Did you test it?

**Comment by div08** [comment]
Hi,
Is there any plan to provide support of this package with Django 5.2, DRF 3.16 and elasticsearch 9.1? 
This package is quite old and not compatible with above mentioned packages.
